apps/backend/src/transaction/entities/transaction.entity.ts

```diff
@@ -35,10 +35,9 @@ export class TransactionEntity {
   location_id: number;
 
   @Column({ type: 'jsonb', nullable: false })
-  transactionJson: Partial<Transaction>;
+  transactionJson?: Transaction;
```

Comment:
Column is required, however we can create an instance of the TransactionEntity in the code and omit this if necessary.